Tests/UnitTests/TestHelpers/TestLogHandler.swift

```diff
@@ -134,7 +134,10 @@ extension TestLogHandler {
             line: line,
             self.messages
         )
-        .toNot(containElementSatisfying(Self.entryCondition(message: message, level: level)))
+        .toNot(
+            containElementSatisfying(Self.entryCondition(message: message, level: level)),
```

Comment:
This provides a better failed assertion message, similar to `verifyMessageWasLogged`.